kinesis/src/main/scala/akka/stream/alpakka/kinesis/KinesisFlowErrors.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.kinesis
+
+import com.amazonaws.services.kinesis.model.PutRecordsResultEntry
+
+import scala.util.control.NoStackTrace
+
+object KinesisFlowErrors {
```

Comment:
If we renamed this `KinesisErrors` I could also put `KinesisSourceErrors` in here as well, but it's up to you if you think they're better off in separate files. Same with settings.